jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/ByteBufferPool.java

```diff
@@ -122,6 +122,21 @@ class Sized extends Wrapper
         private final boolean _direct;
         private final int _size;
 
+        /**
+         * Get a sized pool for any arbitrary ByteBufferPool.
+         *
+         * @param pool the pool
+         * @param direct {@code true} for direct buffers (if wrapping pool)
+         * @param size The specified size in bytes of the buffer, or -1 for a default
+         * @return the pool as a sized pool
+         */
+        public static ByteBufferPool.Sized as(ByteBufferPool pool, boolean direct, int size)
```

Comment:
I don't think I like this new helper as it takes parameters that could be ignored without any hint.

Reply:
Removed